Adds AM_MAINTAINER_MODE to configure.ac so the timestamp of the
automake files won't be updated

Index: debian-mutt-sidebar/configure.ac
===================================================================
--- debian-mutt-sidebar.orig/configure.ac	2007-08-11 16:42:36.000000000 +0200
+++ debian-mutt-sidebar/configure.ac	2007-08-11 16:42:49.000000000 +0200
@@ -9,6 +9,7 @@ AM_CONFIG_HEADER([config.h])
 
 mutt_cv_version=`cat $srcdir/VERSION`
 AM_INIT_AUTOMAKE(mutt, $mutt_cv_version)
+AM_MAINTAINER_MODE
 AC_SUBST([CONFIG_STATUS_DEPENDENCIES], ['$(top_srcdir)/VERSION'])
 
 AC_GNU_SOURCE
